### ISSUE TITLE
CRAN errors reproduced in sf/sp/rgdal/raster revdeps

### ISSUE THREAD

**Issue by rsbivand** [joelgombin/concaveman#10]
Please try to replicate the errors with PROJ >= 6, GDAL >= 3, and CRAN sf:

```
* checking examples ... ERROR
Running examples in ‘concaveman-Ex.R’ failed
The error most likely occurred in:

> ### Name: concaveman
> ### Title: concaveman: A very fast 2D concave hull algorithm.
> ### Aliases: concaveman concaveman-package concaveman concaveman.matrix
> ###   concaveman.sf concaveman.SpatialPoints
> 
> ### ** Examples
> 
> data(points)
> polygons <- concaveman(points)
Error: All columns in a tibble must be vectors.
✖ Column `geometry` is a `sfc_POINT/sfc` object.
Backtrace:
     █
  1. ├─concaveman::concaveman(points)
  2. └─concaveman:::concaveman.sf(points)
  3.   └─`%>%`(...)
  4.     ├─base::withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
  5.     └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
  6.       └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
  7.         └─concaveman:::`_fseq`(`_lhs`)
  8.           └─magrittr::freduce(value, `_function_list`)
  9.             ├─base::withVisible(function_list[[k]](value))
 10.             └─function_list[[k]](value)
 11.               ├─dplyr::summarise(...)
 12.               └─dplyr:::summarise.data.frame(...)
 13.                 ├─base::as.data.frame(summarise(tbl_df(.data), ...))
 14.                 ├─dplyr::summarise(tbl_df(.data), ...)
 15.                 └─dplyr::tbl_df(.data)
 16.                   ├─tibble::as_tibble(d
Execution halted
```
Please refer to **sf** PROJ issues for links to containers, or install PROJ/GDAL yourself to check.

Probably a **dplyr**/**tidyverse** non-backwards-compatible problem, mute example if easier.

**Comment by rsbivand** [comment]
Not just a **dplyr** problem. After following @edzer s comment https://github.com/r-spatial/sf/issues/1314#issuecomment-606886809, and installing **dplyr** github/master, the next error is:

```
* checking examples ... ERROR
Running examples in ‘concaveman-Ex.R’ failed
The error most likely occurred in:

> ### Name: concaveman
> ### Title: concaveman: A very fast 2D concave hull algorithm.
> ### Aliases: concaveman concaveman-package concaveman concaveman.matrix
> ###   concaveman.sf concaveman.SpatialPoints
> 
> ### ** Examples
> 
> data(points)
> polygons <- concaveman(points)
> plot(points)
> plot(polygons, add = TRUE)
Error in plot.sf(polygons, add = TRUE) : 
  plotting list-columns not supported
Calls: plot -> plot.sf
Execution halted
```
So the first problem was a **dplyr** problem, thhe second is your problem with released **sf** 0.9-0.

**Comment by joelgombin** [comment]
fixed by upcoming CRAN release.
